### PR TITLE
fix(CI/CD): update pipeline.ci routine in CD step

### LINKED ISSUE #2460: Master build is broken
The recent changes to the Jenkins file broke the master build.
Master build fails with 
```
java.lang.NoSuchMethodError: No such DSL method 'ci' found among steps ...
```
This line needs to be updated https://github.com/fabric8-ui/fabric8-planner/blob/master/Jenkinsfile#L39

### PR DESCRIPTION
**Pull request by debloper**
Closes #2460

### CODE REVIEW

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2461-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2461-fabric8-planner.badger.fabric8.io

File: Jenkinsfile [hunk 1]
```diff
@@ -35,9 +35,7 @@ fabric8UITemplate{
                     sh "git pull"
                     sh "git remote set-url origin git@github.com:${project}.git"
 
-                    container('ui'){
```
Comment (jarifibrahim) [inline Jenkinsfile]: Why did you remove this line?

Comment (pranavgore09) [reply Jenkinsfile]: `release.groovy` already adds the container('ui').
this change looks good

Comment (jarifibrahim) [reply Jenkinsfile]: Oh, okay. I missed that.

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2461-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2461-fabric8-planner.badger.fabric8.io
